app/views/pages/eligibility/directorEligibility.scala.html

```diff
@@ -10,6 +10,16 @@
     )
     <h1 class="form-title heading-large" id="pageHeading">@Messages("pages.directorEligibility.heading")</h1>
```

Comment:
i think looking at the prototype this should have the class heading-xlarge and not heading-large